packages/wonder-blocks-labeled-field/src/components/__tests__/labeled-field.test.tsx

```diff
@@ -0,0 +1,225 @@
+import * as React from "react";
```

Comment:
This file is copied from the [FieldHeading tests](https://github.com/Khan/wonder-blocks/blob/f451591b361a60883c7a86e33d2a2d117a05fa41/packages/wonder-blocks-form/src/components/__tests__/field-heading.test.tsx). References to field heading were renamed to labeled field